scripts/sysman/virtualFunctionManagement.yml

```diff
@@ -0,0 +1,252 @@
+#
+# Copyright (C) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+#
+# See YaML.md for syntax definition
+#
+--- #--------------------------------------------------------------------------
+type: header
+desc: "Intel $OneApi Level-Zero Sysman Extension APIs for Virtual Function Management Properties"
+version: "1.9"
+--- #--------------------------------------------------------------------------
+type: macro
+desc: "Virtual Function Management Extension Name"
+version: "1.9"
+name: $S_VIRTUAL_FUNCTION_MANAGEMENT_EXP_NAME
+value: '"$XS_extension_virtual_function_management"'
+--- #--------------------------------------------------------------------------
+type: enum
+desc: "Virtual Function Management Extension Version(s)"
+version: "1.9"
+name: $s_vf_management_exp_version_t
+etors:
+    - name: "1_0"
+      value: "$X_MAKE_VERSION( 1, 0 )"
+      desc: "version 1.0"
+--- #--------------------------------------------------------------------------
+type: enum
+desc: "Virtual function memory type flags"
+version: "1.9"
+class: $sVFManagement
+name: $s_vf_info_mem_type_exp_t
+etors:
+    - name: MEM_TYPE_SYSTEM
+      value: "0"
+      desc: "System memory"
+    - name: MEM_TYPE_DEVICE
+      desc: "Device local memory"
+--- #--------------------------------------------------------------------------
+type: enum
+desc: "Virtual function utilization flag bit fields"
+version: "1.9"
+class: $sVFManagement
+name: $s_vf_info_util_exp_flag_t
```

Comment:
is this an enum or flags? If it is an enum, please use a different name. If it is flags, then use `$s_vf_info_util_exp_flags_t`.